robottelo/utils/ohsnap.py

```diff
@@ -2,13 +2,21 @@
 import requests
 from box import Box
 from packaging.version import Version
+from wait_for import wait_for
 
 from robottelo import constants
 from robottelo.exceptions import InvalidArgumentError
 from robottelo.exceptions import RepositoryDataNotFound
 from robottelo.logging import logger
 
 
+def ohsnap_response_hook(r, *args, **kwargs):
+    """Requests response hook callback function that processes the response"""
```

Comment:
Expand on docstrings for parameters atleast for `r`.

Reply:
done